configurations/config-1/locations/location-1/objects/object-1.json

```diff
@@ -1,7 +1,9 @@
 {
   "tag": "object-1",
   "assetPath": "assets/stone.png",
-  "type": "",
-  "x": 150,
-  "y": 200
+  "type": "collectible",
+  "position": {
+    "row": 200,
+    "col": 150
+  }
```

Comment:
pozycja poza mapą (20x20)

Reply:
Omg, dlatego mi obiektów nie pokazywało, ale <333333

Reply:
to naprawia u mnie obiekty, playera dalej brak